models/Walk_2018/README.md

```diff
@@ -1,3 +1,19 @@
 Data from L. Walk, I. Tamborra, H.-T. Janka, A. Summa, [Identifying rotation in SASI-dominated core-collapse supernovae with a neutrino gyroscope](https://arxiv.org/abs/1807.02366), Phys. Rev. D 98:123001, 2018.
 
 Data taken from the [Garching Supernova archive](https://wwwmpa.mpa-garching.mpg.de/ccsnarchive/data/Walk2018/) for 15Msun progenitor, with permission to use them within snewpy.
+
+
+Non Rotating
+Strong: (3.0166, 0.19636) dir1
+Intermediate: (2.1242, 1.0889) dir2
+Weak: (1.6601, 1.6601) dir3
```

Comment:
> Thanks for adding these models! A couple things I’ve noticed:
> 
> * You’ve included commits from #230 in this PR; so we should make sure to merge that one first.
> * `models/Walk_2018/s15.0c_3D_fastrot_dir2_LS220_nux` appears to be an empty file?
> * We’ve previously (#161, #210) had some duplicate time bins in Walk model files. Could you check for duplicate lines in these files and remove them if necessary? (See e.g. [this sample code snippet](https://github.com/SNEWS2/snewpy/pull/161#issuecomment-1005621087) to save some time.)

- I will check the empty file
- I already checked for duplicated lines in the new files before adding them, I will check the old ones as well

Reply:
> Do those numbers correspond to theta and phi? It would be good to add a sentence of explanation to make this clearer.

yes they do, I will make this more clear